scripts/dist-management/pack-and-upload.sh

```diff
@@ -2,6 +2,6 @@
 
 declare -r HASH="$(git rev-parse --verify HEAD)"
 
-zip -R -y "$HASH" './packages/**/dist/**/*' dist/**/* './packages/**/dist/*' dist/*
+find packages -type f -path '*dist*' | zip "$HASH" -@
```

Comment:
Narrowing `dist` to be a directory name (as opposed to part of another word like "distance") feels a bit better. Also won't this risk matching `dist` inside of nested `node_modules` folders (if any exist)?
```suggestion
find packages -type f -path '*/dist/*' | zip "$HASH" -@
```

Reply:
I'm narrowing it down to things under `packages`. If this is run in CI it should be fine.